### PR TITLE
Handle commit formatting with no login value

### PR DESCRIPTION
**Pull request by ksnavely**
I ran into this early today. Here I handle the case where the value of the 'login' field is not available / None.

### CODE REVIEW

File: src/cirrus/github_tools.py [hunk 1]
```diff
@@ -142,14 +142,18 @@ def get_commit_msgs(owner, repo, since_sha, token=None):
...
     result = []
     for commit in resp.json():
+
+        blame = None
         if commit.get("committer"):
-            committer = commit['committer']['login']
-        else:
+            blame = commit['committer'].get('login')
+
+        if blame is None:
```
Comment (shudgston) [inline src/cirrus/github_tools.py]: This can become an `else`

**Comment by evansde77** [comment]
+1

File: src/cirrus/github_tools.py (hunk 1 above)
Comment (ksnavely) [inline src/cirrus/github_tools.py]: The reason have it as a None check, is because it also handles the case that `'login` is None.

Comment (shudgston) [inline src/cirrus/github_tools.py]: Disregard

**Comment by shudgston** [comment]
:+1:

**Pull request merged**